### DISCUSSION TITLE
Beta Update v3.0.13

### DISCUSSION THREAD

**Discussion by IceHacks** [IceHacks/SurvivCheatInjector#864] [Announcements]
Hola.

Permanently patched that _pesky_ Surviv update bug. The cheat should now only break is Kong actually does stuff <sup>(or if another bug appears)</sup>. Also made `autoShoot` and `useBestGun` work better together (less of the artificial delay after switching).

Too **lazy** to transfer the files over here, so you can go download them from the [auto-update repo](https://github.com/x89-h/x12-update/releases/tag/v3.0.13). That auto-update repo **will change** to this SurvivCheatInjector repo **once `v3` is out of beta**.

If you haven't already guessed: I don't play surviv, so don't expect too many updates.

<hr>

Once again, for the reading impaired <sup>(and slow kids)</sup>:

### [> DOWNLOAD HERE <](https://github.com/x89-h/x12-update/releases/tag/v3.0.13)

<hr /><em>This discussion was created from the release <a href='https://github.com/IceHacks/SurvivCheatInjector/releases/tag/v3.0.13'>Beta Update v3.0.13</a>.</em>

**Comment by AZ9048** [comment]
Good work! The damn green screen no longer shows up when i enter the game. Thanks you very much!

**Reply by drdaen** [reply]
how do u implement the hacks?

**Comment by TylerWhat** [comment]
nice

**Comment by khangmoi** [comment]
đéo có

**Comment by ssgg12** [comment]
3.0.14 is worst update ,the gun is shaking strong and accuracy is just like 45% i love the 3.0.10 ;(

**Reply by viridianknight7** [reply]
Thanks for your valued opinion! Nobody cares though, so please shove it up your ass!

**Reply by russyboy1** [reply]
oof xd

**Comment by iMadeThisAccToHack** [comment]
How do i download on mac? it doesn't work

**Reply by viridianknight7** [reply]
oops sorry about that! youll have to find another way to hack on crap OS I guess. Maybe try not using an apple device? that might work.

**Comment by Player001WST** [comment]
What I Need Is a File, Not a Photo

**Comment by Sparky8314** [comment]
Can someone help me? I can't find how to use the hack. I don't know which one to download sicne there are a lot of links.

**Comment by JustPhilYT** [comment]
How to do quickswitching

**Comment by KenshiroYTB** [comment]
I know this is crazy and it was removed long time ago but, can you add or remake the FollowBot? Cause it would be funny for me if i use it to follow a teammate Lol

**Comment by itmightbedeeply** [comment]
This mod doesn't work on mac. When you start it up, it just shows the surviv.io layout. No game no nothing, just stuck there. Any help?

**Reply by russyboy1** [reply]
clear cache

**Reply by russyboy1** [reply]
dont use mac to play surviv cheats

**Comment by getkilledbyme** [comment]
how do you get to the real file some are images and many more and i cant see one of the real cheat injector.

**Reply by russyboy1** [reply]
???

**Reply by russyboy1** [reply]
what version are you using

**Reply by russyboy1** [reply]
there shouldnt be any images???

**Reply by russyboy1** [reply]
are you using the client(3.0.13) or pre 3.0 versions(e.g 2.1.7)

**Reply by russyboy1** [reply]
pre 3.0 versions WILL contain images like this
[
![Screenshot_168](https://user-images.githubusercontent.com/107330802/176169626-ceabd1fc-a571-48fd-813f-d5c69fdb4a63.png)
](url)

**Comment by XxXJEFFXDXxX** [comment]
<img width="758" alt="屏幕截图 2022-07-15 162458" src="https://user-images.githubusercontent.com/102808200/179243589-9f3f2e1d-f512-4398-87a9-b372f113f4fc.png">
I cannot log in with my google account. Help!

**Comment by oldversionworks** [comment]
bro version 2.1.7 works perfectly fine for me why yall complaining about a green screen of death

**Reply by russyboy1** [reply]
yeah boi same, people just dont care about me and they just spam

**Reply by silentaim** [reply]
2.1.7 the best version fr

**Reply by russyboy1** [reply]
ys, sniper switch works perfect

**Reply by JIAJIAPNXA** [reply]
2.1.7. is sure better than 3.0.13. cause in 2.1.7. 1, pan hero works. 2 auto chase works only when you press attack button,in 3,0,13 you wont escape from enemy before you kill him 3, no auto crate break, 4 sniper switch quite better. it can change to melee if you have 1 sniper. and will change to another if you have two snipers. 5. enemies have flashlight too. for example you can use it to kill red commandor just bypassing his super 90 flashlight, also it helps to evade bullets.

**Comment by hehehiha** [comment]
i cant login with my google account i need help @IceHacks

**Reply by SurvivGuy00** [reply]
Same . Also it says Failed joining game. please help @IceHacks
